code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,153 @@
+			temp.pixel_x = Clamp(text2num(params_list["icon-x"]) - clamp_x, -clamp_x, clamp_x)
+			temp.pixel_y = Clamp(text2num(params_list["icon-y"]) - clamp_y, -clamp_y, clamp_y)
+
+		overlays += temp
+
+	clicked = new/icon(src.icon, src.icon_state, src.dir)
+
+/obj/item/mounted/frame/trophy_mount/examine(mob/user)
+	..()
+	if(held_item)
+		held_item.examine(user, show_name = FALSE)
+
+/obj/item/mounted/frame/trophy_mount/attackby(obj/item/weapon/W, mob/user, params)
+	..()
+	params_list = params2list(params)
+	if(iswrench(W))
+		return
+	mount_item(W, user)
+
+/obj/item/mounted/frame/trophy_mount/AltClick(mob/user)
+	var/obj/item/I = user.get_active_hand()
+	if(I && I != src)
+		params_list = list()
+		mount_item(I, user)
+
+/obj/item/mounted/frame/trophy_mount/proc/mount_item(obj/item/weapon/W, mob/user)
+	if(held_item)
+		to_chat(user, "This [initial(name)] already has \a [held_item] mounted on it.")
+		return
+	if(user.drop_item(W, src))
+		user.visible_message("\The [user] mounts \the [W] onto \the [src].", "You mount \the [W] onto \the [src].")
+		held_item = W
+		update_icon()
+
+
+/obj/item/mounted/frame/trophy_mount/attack_self(mob/user)
+	if(held_item)
+		var/obj/item/I = held_item
+		held_item.forceMove(get_turf(src))
+		user.put_in_hands(held_item)
+		held_item = null
+		params_list = list()
+		update_icon()
+		user.visible_message("\The [user] removes \the [I] from \the [src].", "You remove \the [I] from \the [src].")
+
+/obj/item/mounted/frame/trophy_mount/do_build(turf/on_wall, mob/user)
+	if(!user.drop_item(src))
+		to_chat(user, "<span class='warning'>You can't let go of \the [src]!</span>")
+		return
+	user.visible_message("\The [user] hangs \the [src] on \the [on_wall].", "You hang \the [src] on \the [on_wall].")
+	add_fingerprint(user)
+	var/obj/structure/trophy_mount/T = new(get_turf(src))
+	T.name = name
+	T.desc = desc
+	if(held_item)
+		held_item.forceMove(T)
+		T.held_item = held_item
+		held_item = null
+		T.params_list = params_list
+	T.overlays += appearance
+	transfer_fingerprints(src, T)
+	var/direction = get_dir(user,on_wall)
+	if(direction & NORTH)
+		T.pixel_y = WORLD_ICON_SIZE
+	if(direction & SOUTH)
+		T.pixel_y = -WORLD_ICON_SIZE
+	if(direction & EAST)
+		T.pixel_x = WORLD_ICON_SIZE
+	if(direction & WEST)
+		T.pixel_x = -WORLD_ICON_SIZE
+	playsound(on_wall, 'sound/items/Deconstruct.ogg', 25, 1)
+	qdel(src)
+
+/obj/structure/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	autoignition_temperature = AUTOIGNITION_WOOD
+	anchored = 1
+	var/obj/item/held_item
+	var/list/params_list
+
+/obj/structure/trophy_mount/attack_hand(mob/user)
+	to_chat(user, "<span class='notice'>\The [src] is mounted securely. You'll need something to pry it off the wall.</span>")
+
+/obj/structure/trophy_mount/attackby(obj/item/weapon/W, mob/user)
+	if(iscrowbar(W))
+		to_chat(user, "You begin prying the [initial(name)] off the wall.")
```

Comment:
`\the`

Reply:
I was told that `\the` only works with embedded references, not embedded variables.

Reply:
Nah it works fine with text too.